docs/hacking/chrome-extension.rst

```diff
@@ -20,7 +20,7 @@ To build and install the Chrome extension:
 
    .. code-block:: bash
 
-      hypothesis-buildext chrome --debug --service 'http://127.0.0.1:5000'
+      hypothesis-buildext chrome --debug --service 'http://127.0.0.1:5000' --bouncer-url "http://127.0.0.1:8000/*"
```

Comment:
I'm not sure I agree, but either way it needs to be updated to reflect the correct parameter now.